### PR TITLE
Release bignumber-2.0.0

### PR DESCRIPTION
**Pull request by cbeams**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -11,7 +11,7 @@
     <packaging>jar</packaging>
     <groupId>org.webjars</groupId>
     <artifactId>bignumber</artifactId>
-    <version>1.3.1-SNAPSHOT</version>
+    <version>2.0.0</version>
```
Comment (jamesward) [inline pom.xml]: This needs to be `2.0.0-SNAPSHOT` since Maven does the actual release.

File: pom.xml [hunk 2]
```diff
@@ -36,12 +36,12 @@
         <url>http://github.com/webjars/bignumber</url>
         <connection>scm:git:https://github.com/webjars/bignumber.git</connection>
         <developerConnection>scm:git:https://github.com/webjars/bignumber.git</developerConnection>
-        <tag>HEAD</tag>
+        <tag>bignumber-2.0.0</tag>
```
Comment (jamesward) [inline pom.xml]: This needs to be left as `HEAD`.

**Comment by cbeams** [comment]
Hey @jamesward, thanks for the quick responses. I've force-pushed changes in accordance with your corrections; should be good to go for the maven release process now.

**Comment by cbeams** [comment]
Awesome. Thanks James. Got a couple brand new web jars on the way after this.

**Comment by jamesward** [comment]
Thanks! This will be released shortly.

**Pull request merged**